pillar/rds-articles.yaml

```diff
@@ -11,6 +11,12 @@
 
 '30274':
     date: '2030-01-01T00:00:00Z'
```

Comment:
@nlisgo, I've kept the date as it previously was, as I can't remember why it was originally like this. Something to do with the test site I think? 
Should I be changing the date for this?

Reply:
Obviously I know in the future that the dates need to correct, I just mean for the purposes of this first check on the test site.

Reply:
> @nlisgo, I've kept the date as it previously was, as I can't remember why it was originally like this. Something to do with the test site I think?
> Should I be changing the date for this?

I believe this was because we did not know the date to use and to force the article to appear at the top of the listing. We should alter it when we want it to appear correctly in the listings.

Reply:
Sure, sounds good, but it's fine as it is now for the purposes of pushing to the test site for review?